auth/auth.js

```diff
@@ -51,6 +52,69 @@ exports.getAuth = async (req, res, next) => {
   }
 }
 
+exports.signAuth=async(req,res)=>
+{
+
+let email=req.body.email;
+ axios.get(`http://localhost:4000/api/v1/clients/${email}`)
```

Comment:
@amandeeprikhi The URLS should be in the config?

Reply:
Use this file config/medicSoftDbMicroserviceURLs.js. It has all the URLs for Mongo Service.

Reply:
This has been fixed.
Resolving this conversation.